src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -196,15 +201,14 @@ private Result<List<CreateAccountCommand>> buildAccountsAndActions(
       }
 
       if (policy.getDeclareLostProcessingFee().isChargeable()) {
-        log.debug("Charging lost item processing fee");
 
+        log.debug("Charging lost item processing fee for set or actual costs");
```

Comment:
Why do we need to change this log at all? Is "Charging lost item processing fee" incorrect now?

Reply:
Moved log to initial condition. But wished to highlight that it works for both states.